pkg/mlsvalidate/service.go

```diff
@@ -133,18 +134,18 @@ func (s *MLSValidationServiceImpl) ValidateKeyPackages(
 
 func makeValidateKeyPackageRequest(
 	keyPackageBytes [][]byte,
-) *svc.ValidateInboxIdKeyPackagesRequest {
+) *svc.ValidateKeyPackagesRequest {
 	keyPackageRequests := make(
-		[]*svc.ValidateInboxIdKeyPackagesRequest_KeyPackage,
+		[]*svc.ValidateKeyPackagesRequest_KeyPackage,
 		len(keyPackageBytes),
 	)
 	for i, keyPackage := range keyPackageBytes {
-		keyPackageRequests[i] = &svc.ValidateInboxIdKeyPackagesRequest_KeyPackage{
+		keyPackageRequests[i] = &svc.ValidateKeyPackagesRequest_KeyPackage{
 			KeyPackageBytesTlsSerialized: keyPackage,
 			IsInboxIdCredential:          true,
 		}
 	}
-	return &svc.ValidateInboxIdKeyPackagesRequest{
+	return &svc.ValidateKeyPackagesRequest{
 		KeyPackages: keyPackageRequests,
 	}
```

Comment:
This is the accompanying code change for https://github.com/xmtp/proto/pull/212. I'm not sure how the auto-proto generation PR passed CI :/